modules/canbus/proto/BUILD

```diff
@@ -13,6 +13,7 @@ proto_library(
         "canbus_conf.proto",
         "chassis.proto",
         "chassis_detail.proto",
+        "ch.proto",
```

Comment:
Could you change it to the fullname to make it more meaningful?

Reply:
Ch is the abbreviation of coolhigh for Development Suite partners, and which is accepted in the branch v3.1_dev, they need to be unified. please review again，think you！